src/management/index.js

```diff
@@ -299,26 +299,15 @@ var ManagementClient = function(options) {
  * @return {Object}   Object containing client information.
  */
 ManagementClient.prototype.getClientInfo = function() {
-  var clientInfo = {
+  return {
```

Comment:
Seems like it would be a good idea to pull this and `AuthenticationClient.prototype.getClientInfo` from the same place...

Reply:
I can extract a new file/class and reference it from both places. Initially I thought on adding this header on a common place as well, instead of having each "set of endpoints module" set it. But I don't want to make a huge refactor of this. I know the work for auto-generating the APIs is eventually going to replace all.